Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -11,10 +11,18 @@ import Foundation
 
 @MainActor
 class DashboardViewModel: ObservableObject {
-    
     let authenticationViewModel: AuthenticationViewModel
     
     @Published var events = [MaintenanceEvent]()
+    @Published var sortOption: SortOption = .custom
+    
+    var sortedEvents: [MaintenanceEvent] {
+        switch sortOption {
+        case .oldestToNewest: events.sorted(by: {$0.date < $1.date })
+        case .newestToOldest: events.sorted(by: { $0.date > $1.date })
```

Comment:
```suggestion
        case .oldestToNewest: events.sorted(by: { $0.date < $1.date })
        case .newestToOldest: events.sorted(by: { $0.date > $1.date })
```

Added one extra space, can this be a trailing closure with just like this `events.sorted { $0.date < $1 }` without the `by` parameter?